internal/tools/changes/cmd/changetool/change.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/internal/tools/changes"
+	"strconv"
+)
+
+var changeParams = struct {
+	module      string
+	changeType  string
+	description string
+	similar     bool
+}{}
+
+var addFlags *flag.FlagSet
+var lsFlags *flag.FlagSet
+
+func init() {
+	addFlags = flag.NewFlagSet("add", flag.ExitOnError)
+	addFlags.StringVar(&changeParams.module, "module", "", "creates a change for the specified module")
+	addFlags.StringVar(&changeParams.changeType, "type", "", "sets the change's type")
+	addFlags.StringVar(&changeParams.description, "description", "", "sets the change's description")
+
+	lsFlags = flag.NewFlagSet("ls", flag.ExitOnError)
+	lsFlags.StringVar(&changeParams.module, "module", "", "filters changes by module")
+}
+
+func changeSubcmd(args []string) error {
+	if len(args) == 0 {
+		usage()
+	}
+
+	changesPath, err := changes.GetChangesPath()
+	if err != nil {
+		return fmt.Errorf("failed to load .changes directory: %v", err)
+	}
+
+	metadata, err := changes.LoadMetadata(changesPath)
+	if err != nil {
+		return fmt.Errorf("failed to load .changes directory: %v", err)
+	}
+
+	switch args[0] {
+	case "add", "new":
+		addFlags.Parse(args[1:])
+		return addCmd(metadata, changeParams.module, changeParams.changeType, changeParams.description)
+	case "ls", "list":
+		lsFlags.Parse(args[1:])
+		return lsCmd(metadata, changeParams.module)
```

Comment:
It might make sense to add this later. Right now the tool doesn't know what packages a module provides, but I think once the release tool part is developer that information should be available.